### PR TITLE
add support for append_paths in module generator

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/tools/module_generator.py [hunk 1]
```diff
@@ -132,6 +132,23 @@ def __init__(self, application, fake=False):
...
+        :param allow_abs: allow providing of absolute paths
+        :param expand_relpaths: expand relative paths into absolute paths (by prefixing install dir)
+        """
+        try:
+            txt = self.prepend_paths(key, paths, allow_abs=allow_abs, expand_relpaths=expand_relpaths)
+        except EasyBuildError as err:
+            raise EasyBuildError(str(err).replace('prepend', 'append'))
+
+        regex = re.compile('^prepend([-_]path)', re.M)
+        return regex.sub(r'append\1', txt)
```
Comment (wpoely86) [inline easybuild/tools/module_generator.py]: I don't really like this.

Comment (boegel) [reply easybuild/tools/module_generator.py]: The alternative is to copy-paste the code for `prepend_paths`, and replace `prepend` with `append`. I'm not a big fan of that approach...

Comment (akesandgren) [reply easybuild/tools/module_generator.py]: It would be a less confusing approach though. This one is slightly ugly in my opinion.

**Comment by boegel** [comment]
@akesandgren @wpoely86 better now?

**Pull request merged**